.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,59 @@
+name: Release Linux
+
+on: 
+  workflow_dispatch:
+    inputs:
+      commit_hash:
+        description: "Release Candidate Commit (example: 5a3ae3)"
+        required: true
+
+jobs:
+  release: 
+    env:
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with: 
+        commit: ${{ github.event.inputs.commit_hash }}
+
+    - name: Install Node
+      uses: actions/setup-node@v2.2.0
+      with:
+        node-version: 16
+
+    - name: Install NPM Dependencies
+      run: npm ci
+
+    - name: Tag Release Commit
+      run: |
+        git config --global user.name "OutlineBot"
+        git config --global user.email "outlinebot@users.noreply.github.com"
+
+        git tag "linux-$(node scripts/get_version.mjs linux)"
```

Comment:
This should yell if the tag already exists, preventing us from accidentally releasing the same version twice